src/Samples.Shared/Managers/LicenseStrings.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace ArcGIS.Samples.Shared.Managers
+{
+    internal static partial class LicenseStrings
```

Comment:
This class might be named something like "LicenseManager" to follow convention of other manager classes in our repo.

Reply:
I'm happy either way, I think in this case `LicenseStrings` is fine because it's a static class holding static strings. 

Either works for me.